### ISSUE TITLE
Fail to create resource file with image. 

### ISSUE THREAD

**Issue by radiy** [dotnet/winforms#251]
* .NET Core Version: (e.g. 3.0 Preview1, or daily build number, use `dotnet --info`)

3.0 Preview1, vs 2019 preview 1.1

* Have you experienced this same bug with .NET Framework?: Yes/No

no

**Problem description:**

Fail to create resource file with image. 

**Actual behavior:** 

Build will fail with error:
1>C:\Users\kvasov2\projects\tmp\res-bug\Form1.resx(123,5): error MSB3103: Invalid Resx file. Could not load file or assembly 'System.Drawing.Common, Version=4.0.1.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies. Reference assemblies should not be loaded for execution.  They can only be loaded in the Reflection-only loader context. (Exception from HRESULT: 0x80131058) Line 123, position 5.

**Expected behavior:**

Successful build.

**Minimal repro:**

* dotnet new winforms
* open vs 2019
* add jpg to Form1.resx
* build

**Comment by danmosemsft** [comment]
Same as https://github.com/Microsoft/msbuild/issues/3311 ? 

At a guess, this is from ResxResourceReader, probably the Assembly.Load that's in ResXDataNode.GetAssemblyQualifiedName.

**Comment by dreddy-work** [comment]
@radiy, are you building it from the VS or from the dotnet core command line?

**Comment by radiy** [comment]
Building from vs

19 дек. 2018 г. 21:56 пользователь "Devendar Reddy Adulla" <
notifications@github.com> написал:

@radiy <https://github.com/radiy>, are you building it from the VS or from
the dotnet core command line?

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/dotnet/winforms/issues/251#issuecomment-448705719>, or mute
the thread

**Comment by lextm** [comment]
The error message only shows when compiling in VS. If running `dotnet build` then everything seems to compile.

However, another bigger issue is that the actual embedded resource contains a string instead of the image, which triggers a runtime exception like,
```
Application: JexusManager.exe
CoreCLR Version: 4.6.27121.3
Description: The process was terminated due to an unhandled exception.
Exception Info: System.InvalidCastException: Unable to cast object of type 'System.String' to type 'System.Drawing.Bitmap'.
   at JexusManager.Main.Properties.Resources.get_connect_16() in F:\JexusManager\JexusManager\Properties\Resources.Designer.cs:line 159
   at JexusManager.MainForm.InitializeComponent() in F:\JexusManager\JexusManager\MainForm.Designer.cs:line 339
   at JexusManager.MainForm..ctor(List`1 files) in F:\JexusManager\JexusManager\MainForm.cs:line 78
   at JexusManager.Program.Main(String[] args) in F:\JexusManager\JexusManager\Program.cs:line 83
```
If you need a sample project, just do the following,

1. `dotnet new winforms` with .NET Core 3.0 SDK Preview 1.
1. Open this project in VS2019 Preview 1 (or 1.1).
1. Add an image to `Resources.resx` under Properties.
1. Save the project and compile in VS.

Then the initial error reported can appear.

Switch to `dotnet build` and it should succeed. And then use a tool like ILSpy to decompile the assembly, and the embedded .resources should show a String Table, which is completely wrong.

**Comment by dreddy-work** [comment]
This issue need to be fixed before we can work on #540

**Comment by merriemcgaw** [comment]
This is a duplicate of Microsoft/msbuild#2221 so I will close it.
